### PR TITLE
feat: add allow auto update schema cmd

### PR DESCRIPTION
**Pull request by nodece**
Signed-off-by: Zixuan Liu <nodeces@gmail.com>

### CODE REVIEW

File: pkg/ctl/namespace/get_is_allow_auto_update_schema.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+)
+
+func GetIsAllowAutoUpdateSchemaCmd(vc *cmdutils.VerbCmd) {
+	var desc cmdutils.LongDescription
+	desc.CommandUsedFor = "Get the whether to allow auto update schema on a namespace"
+	desc.CommandPermission = "This command requires tenant admin permissions"
+
+	var examples []cmdutils.Example
+	examples = append(examples, cmdutils.Example{
+		Desc:    "Get allow auto update update schema on a namespace",
```
Comment (zymap) [inline pkg/ctl/namespace/get_is_allow_auto_update_schema.go:34]:
```suggestion
		Desc:    "Get the whether to allow auto update schema on a namespace",
```

File: pkg/ctl/namespace/get_is_allow_auto_update_schema.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	}
+
+	admin := cmdutils.NewPulsarClient()
+	result, err := admin.Namespaces().GetIsAllowAutoUpdateSchema(*ns)
+	if err == nil {
+		if result != nil {
+			vc.Command.Println(strconv.FormatBool(*result))
+		} else {
+			vc.Command.Println(result)
+		}
```
Comment (zymap) [inline pkg/ctl/namespace/get_is_allow_auto_update_schema.go:63]:
```suggestion
			vc.Command.Println(strconv.FormatBool(*result))
```

Looks like printing the result directly is enough

File: pkg/ctl/namespace/set_is_allow_auto_update_schema.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+)
+
+func SetIsAllowAutoUpdateSchemaCmd(vc *cmdutils.VerbCmd) {
+	var desc cmdutils.LongDescription
+	desc.CommandUsedFor = "Set the whether to allow auto update schema on a namespace"
+	desc.CommandPermission = "This command requires tenant admin permissions"
+
+	var examples []cmdutils.Example
+	examples = append(examples, cmdutils.Example{
+		Desc:    "Enable allow auto update update schema on a namespace",
```
Comment (zymap) [inline pkg/ctl/namespace/set_is_allow_auto_update_schema.go:34]:
```suggestion
		Desc:    "Enable automatically update schema on a namespace",
```

File: pkg/ctl/namespace/set_is_allow_auto_update_schema.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+		Command: "pulsarctl namespaces set-is-allow-auto-update-schema --enable (namespace-name)",
+	})
+	examples = append(examples, cmdutils.Example{
+		Desc:    "Disable allow auto update update schema on a namespace",
```
Comment (zymap) [inline pkg/ctl/namespace/set_is_allow_auto_update_schema.go:38]:
```suggestion
		Desc:    "Disable automatically update update schema on a namespace",
```

File: pkg/ctl/namespace/set_is_allow_auto_update_schema.go [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+		desc.CommandUsedFor,
+		desc.ToString(),
+		desc.ExampleToString())
+
+	var (
+		enable  bool
+		disable bool
+	)
+	vc.FlagSetGroup.InFlagSet("IsAllowAutoUpdateSchema", func(set *pflag.FlagSet) {
+		set.BoolVar(&enable, "enable", false, "enable allow auto update schema")
```
Comment (zymap) [inline pkg/ctl/namespace/set_is_allow_auto_update_schema.go:54]:
```suggestion
		set.BoolVar(&enable, "enable", false, "enable automatically update schema")
```

File: pkg/ctl/namespace/set_is_allow_auto_update_schema.go [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+		set.BoolVar(&disable, "disable", false, "disable allow auto update schema")
```
Comment (zymap) [inline pkg/ctl/namespace/set_is_allow_auto_update_schema.go:55]:
```suggestion
		set.BoolVar(&disable, "disable", false, "disable automatically update schema")
```

**Review by zymap — APPROVED** [review]
LGTM

File: pkg/ctl/namespace/get_is_allow_auto_update_schema.go (hunk 2 above)
Comment (nodece) [reply pkg/ctl/namespace/get_is_allow_auto_update_schema.go:63]: We need to check nil pointer.

**Pull request merged**